packages/jest-jasmine2/src/setup-jest-globals.js

```diff
@@ -42,7 +42,7 @@ const addSuppressedErrors = result => {
   }
 };
 
-const addAssertionErrors = result => {
+function addAssertionErrors(result) {
```

Comment:
You might wonder why of this change, node 4 has a bug where it guess this function name to be "result", which is wrong.
Changing it to a function gives the function the correct name, so no more guessing, and it pass the tests correctly.